### PR TITLE
Eschew Docker usage in Drone tests

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

File: mltsp/tests/test_custom_feats.py [hunk 1]
```diff
@@ -89,6 +89,8 @@ def test_execute_functions_in_order():
 
 def test_docker_installed():
```
Comment (stefanv) [inline mltsp/tests/test_custom_feats.py]: Let's use the "skipif" decorator here

**Comment by stefanv** [comment]
How about not touching the config file, and adding two Makefile targets: one for testing with, the other without Docker.

**Comment by acrellin** [comment]
I was just thinking that - thanks Stefan

**Comment by acrellin** [comment]
@stefanv, what do you think?

File: mltsp/tests/test_custom_feats.py [hunk 2]
```diff
@@ -87,6 +88,8 @@ def test_execute_functions_in_order():
     assert(all(x in feats_list[0] for x in ["a", "l", "o"]))
```
Comment (stefanv) [inline mltsp/tests/test_custom_feats.py]: Suggestion:

```no_docker = (os.getenv("MLTSP_NO_DOCKER") == "1")```

and then later on

```@skipIf(no_docker)```

Also, let's use the slightly more sophisticated implementation of ``skipif``:

```
from numpy.testing import decorators as dec

@dec.skipif(no_docker, "Docker testing turned off")
...
```

**Comment by acrellin** [comment]
Could also put `no_docker` in the `cfg` to clean it up even further...

**Pull request merged**